SSINS/tests/test_INS.py

```diff
@@ -459,4 +469,4 @@ def test_set_weights_square_array(cross_testfile):
     ins.set_ins_data_params()
 
     assert ins.weights_square_array is not None
-    assert np.array_equal(ins.metric_ms, copy_ins.metric_ms) # check that the goods are intact
\ No newline at end of file
+    assert np.array_equal(ins.metric_ms, copy_ins.metric_ms) # check that the goods are intact
```

Comment:
I'm having trouble understanding from the diff what happened here. Was there an empty line with a space in it or something?